controller/script/simulate-proxy/main.go

```diff
@@ -101,15 +193,15 @@ var (
 	// These values are one order of magnitude greater than the controller's
```

Comment:
realizing this isn't part of your change, but would you mind removing `ports = []uint32{3333, 6262}` above? it's unused.